src/components/dateTimePickerInput.js

```diff
@@ -28,287 +28,265 @@
       validationAfterMaxValue,
       validationInvalidValue,
       margin,
-      helperText = [''],
+      helperText,
```

Comment:
will useText be able to handle undefined?

Reply:
It should, its a helper provided from B-helpers. Handling `undefined` should be part of the helper